Proyecto final Modulo 2

VanessaVillegas opened the pull request:
Adjunto el proyecto final del Modulo 2[](url
[VV-project.pdf](https://github.com/UniversidadEAFIT/compubiol_course/files/8517937/VV-project.pdf)
)
[Nucleotide alignment.geneious.zip](https://github.com/UniversidadEAFIT/compubiol_course/files/8518205/Nucleotide.alignment.geneious.zip)

jcalvarezlab committed:
Delete datos5.cvs

jcalvarezlab committed:
Delete datos6.cvs

jcalvarezlab committed:
Delete lista_mercado.tsv

jcalvarezlab committed:
Add files via upload

Solución Simon

jcalvarezlab committed:
Create notas.rm

jcalvarezlab committed:
Add files via upload

Sin

jcalvarezlab committed:
Update notas.rm

jcalvarezlab committed:
Update readme.rm

jcalvarezlab committed:
Update readme.rm

jcalvarezlab committed:
Update notas.rm

jcalvarezlab committed:
Create sample.rm

jcalvarezlab committed:
Add files via upload

jcalvarezlab committed:
Create sample

jcalvarezlab committed:
Add files via upload

jcalvarezlab committed:
Create sample

jcalvarezlab committed:
Add files via upload

vcaromi committed:
Trabajo final

jcalvarezlab committed:
Merge pull request #89 from vcaromi/master

Trabajo final

smazoza98 committed:
Add files via upload

Trabajo final Santiago Mazo

davidcarmbe committed:
Trabajo final

megatiky committed:
Add files via upload

jcalvarezlab committed:
Merge pull request #92 from davidcarmbe/master

Trabajo final

jcalvarezlab committed:
Merge pull request #91 from smazoza98/master

Trabajo

jcalvarezlab committed:
Merge pull request #93 from megatiky/master

Add files via upload

asanch73 committed:
Add files via upload

jcalvarezlab committed:
Merge pull request #96 from asanch73/master

Add files via upload

jcalvarezlab committed:
Create readme.rm

jcalvarezlab committed:
Create readme.rm

jcalvarezlab committed:
Add files via upload

20200127

Commit:
notebookactualizado

Commit:
genomaausar

Commit:
basicoactualizado

Commit:
äctualizado con la clase

jcalvarezlab committed:
Create readme.rm

jcalvarezlab committed:
Add files via upload

jcalvarezlab committed:
Add files via upload

jcalvarezlab committed:
Delete 20221/LinuxII directory

jcalvarezlab committed:
Add files via upload

jcalvarezlab committed:
Add files via upload

jcalvarezlab committed:
Add files via upload

weedcentipede committed:
Create README.txt

weedcentipede committed:
Add files via upload

jcalvarezlab committed:
Merge pull request #97 from OnlyHigh/master

Test class

amaliamunera committed:
Create amaliamunera

amaliamunera committed:
Delete amaliamunera

amaliamunera committed:
Create TEST

aperezmunoz8 committed:
Create Ejemplo

mmontoyad committed:
Create test

sgomezt1 committed:
Create test

mrodriguel committed:
Create Test

MJPinoH committed:
Create test

jlozano910 committed:
Create test

aperezmunoz8 committed:
Add files via upload

Spatinob committed:
Create Patiño

jptobonv committed:
Create test

amaliamunera committed:
Add files via upload

MJPinoH committed:
Add files via upload

jlozano910 committed:
Add files via upload

estefaniacb committed:
Create test

MJPinoH committed:
Delete test

Spatinob committed:
Add files via upload

aperezmunoz8 committed:
Delete Alejandro Pérez M directory

Brian02031 committed:
Create Brian Hincapie

estefaniacb committed:
Add files via upload

Spatinob committed:
Delete Patiño

amaliamunera committed:
Update TEST

aperezmunoz8 committed:
Create Ejercicio1

FedericoRamirez16 committed:
Create test

mmontoyad committed:
Add files via upload

jptobonv committed:
Create test1

aperezmunoz8 committed:
Add files via upload

Brian02031 committed:
Create test

mrodriguel committed:
Add files via upload

Brian02031 committed:
Add files via upload

mrodriguel committed:
Add files via upload

mecheverrd committed:
Create test

nbetancurb committed:
Create Tarea1-ejecutable.sh

mrodriguel committed:
Delete Task1_ManuelaRodriguez.sh

sgomezt1 committed:
Delete test

mrodriguel committed:
Add files via upload

mrodriguel committed:
Delete Test

jlozano910 committed:
Add files via upload

amaliamunera committed:
Add files via upload

sgomezt1 committed:
Add files via upload

jptobonv committed:
Add files via upload

hola javi acá te adjunto el ejercicio

MJPinoH committed:
Add files via upload

FedericoRamirez16 committed:
Add files via upload

Los que tienen un "v2" se refieren a una segunda versión del reporte 1 (explorando mejoras al proceso)

sgomezt1 committed:
Create test

Spatinob committed:
Add files via upload

mecheverrd committed:
Add files via upload

Buenas tardes javi, te adjunto el ejecutable y el reporte de la actividad

sgomezt1 committed:
Add files via upload

Hola Javi buenas tardes,

Adjunto el reporte de la tarea 1 y el archivo ejecutable.

jcalvarezlab committed:
Add files via upload

Spatinob committed:
Add files via upload

mmontoyad committed:
Add files via upload

jcalvarezlab committed:
Merge pull request #106 from mrodriguel/master

Entrega trabajo de clase Manuela Rodriguez

jcalvarezlab committed:
Merge pull request #105 from nbetancurb/patch-3

Create Tarea1-ejecutable.sh

jcalvarezlab committed:
Merge pull request #104 from estefaniacb/master

Tarea 8/02/2022

jcalvarezlab committed:
Merge pull request #103 from aperezmunoz8/master

Ejercicio1_APM

jcalvarezlab committed:
Merge pull request #102 from amaliamunera/master

Pull request Amalia Múnera

jcalvarezlab committed:
Merge pull request #101 from MJPinoH/master

Tarea biologia computacional

jcalvarezlab committed:
Merge pull request #100 from Spatinob/master

Trabajos Sebastian Patiño

jcalvarezlab committed:
Merge pull request #99 from jlozano910/master

Entrega 1

jcalvarezlab committed:
Merge pull request #98 from mmontoyad/master

Task1_melissa

jcalvarezlab committed:
Merge pull request #107 from Brian02031/master

Pull Request GithubPull

jcalvarezlab committed:
Merge pull request #108 from FedericoRamirez16/master

tarea linux I

scagudeloc committed:
Create Test

scagudeloc committed:
Add files via upload

juanjo255 committed:
carpeta con mi nombre y primera tarea para pull request

jcalvarezlab committed:
Merge pull request #109 from scagudeloc/master

Pull request actividad

jcalvarezlab committed:
Merge pull request #110 from juanjo255/master

carpeta con mi nombre y primera tarea para pull request

VanessaVillegas committed:
Create Tarea_1

jcalvarezlab committed:
Merge pull request #111 from VanessaVillegas/master

Create Tarea_1

jcalvarezlab committed:
Merge pull request #112 from jptobonv/master

pull request ejercicio JPT

mecheverrd committed:
Add files via upload

mecheverrd committed:
Delete nd2 nuestro.png

mecheverrd committed:
Delete nd2.png

mecheverrd committed:
Delete rag1.png

mecheverrd committed:
Delete rag1 nuestro.png

mecheverrd committed:
Merge branch 'UniversidadEAFIT:master' into master

mecheverrd committed:
Se agregó el archivo de la tarea final 

:)

sgomezt1 committed:
Add files via upload

sgomezt1 committed:
Delete Task1_SebastianGomez.sh

jcalvarezlab committed:
Merge pull request #114 from sgomezt1/master

Pull request Github

jcalvarezlab committed:
Merge pull request #113 from mecheverrd/master

Tarea 2 Slurm blasting

scagudeloc committed:
Add files via upload

camilogarciabotero commented:
@VanessaVillegas el PR quedo raro, no entiendo porque aparecen todas las contribuciones de personas de hace rato... Cuando se hace el PR es a la rama "master" o "main", no a esa que dice test... Además lo que pones es un archivo adjunto al comentario. No es algo que montes propiamente al repositorio y busques agregar al conjunto de archivos

jptobonv committed:
Add files via upload

jcalvarezlab committed:
Create readme.rd

jcalvarezlab committed:
Create readme

jcalvarezlab committed:
Add files via upload

Update 2023

jcalvarezlab committed:
Add files via upload

jcalvarezlab committed:
Add files via upload

jcalvarezlab committed:
Add files via upload

jcalvarezlab committed:
Add files via upload

jfrancoc14 committed:
Add files via upload

La tarea

jpmerchanc committed:
Create jpmerchan

hola

mlopezg11 committed:
Create MarianaLópez

dagomezs1 committed:
Create Alejo

sharajg committed:
Create Sjaramillo

BIGNATURE3 committed:
Create AndreaC

Hola

Eled07 committed:
Create Eduardo

Hola

amvidalc committed:
Create VidalAnaMaria

Hola!

sgiraldos3 committed:
Create sgiraldos3

jcalvarezlab committed:
Merge pull request #132 from sgiraldos3/master

Create sgiraldos3

jcalvarezlab committed:
Merge pull request #125 from jpmerchanc/master

Create jpmerchan

jcalvarezlab committed:
Merge pull request #126 from mlopezg11/master

Create MarianaLópez

jcalvarezlab committed:
Merge pull request #124 from jfrancoc14/master

Add files via upload

jcalvarezlab committed:
Merge pull request #127 from BIGNATURE3/master

Create AndreaC

jcalvarezlab committed:
Merge pull request #128 from dagomezs1/master

Create Alejo

jcalvarezlab committed:
Merge pull request #129 from sharajg/master

Create Sjaramillo

jcalvarezlab committed:
Merge pull request #130 from Eled07/master

Create Eduardo

JPDCjp committed:
Create JuanDurango

jcalvarezlab committed:
Merge pull request #131 from amvidalc/master

Create VidalAnaMaria

jcalvarezlab committed:
Merge pull request #133 from JPDCjp/patch-1

Create JuanDurango

sharajg committed:
Add files via upload

mlopezg11 committed:
Add files via upload

mlopezg11 committed:
Delete MarianaLópez

jcalvarezlab committed:
Merge pull request #134 from sharajg/master

Add files via upload

Eled07 committed:
Add files via upload

sharajg committed:
Add files via upload

mlopezg11 committed:
Merge branch 'UniversidadEAFIT:master' into master

Eled07 committed:
Add files via upload

BIGNATURE3 committed:
Add files via upload

Carpeta de trabajos Andrea Carrillo Biología computacional

jcalvarezlab committed:
Merge pull request #135 from Eled07/master

Task1

JPDCjp committed:
Add files via upload

jcalvarezlab committed:
Add files via upload

Script for global alignments shows matrix and alignment. Only for learning

AwkwardDCT committed:
Create danielacalle

jfrancoc14 committed:
Add files via upload

mlopezg11 committed:
Add files via upload

mlopezg11 committed:
Merge branch 'UniversidadEAFIT:master' into master

mlopezg11 committed:
Update BIO487_EjercicioBlast_Preguntas_MLópez.rtf

anamariavc committed:
Create VidalAnaMaria_works

Mi carpeta de trabajos

sharajg committed:
Add files via upload

sharajg committed:
Add files via upload

mlopezg11 committed:
Add files via upload

jcalvarezlab committed:
Merge pull request #121 from scagudeloc/master

Add files via upload

jcalvarezlab committed:
Merge pull request #122 from jptobonv/master

Add files via upload

jcalvarezlab committed:
Merge pull request #137 from mlopezg11/master

Ejercicio blasting

jcalvarezlab committed:
Merge pull request #138 from anamariavc/patch-1

Create VidalAnaMaria_works

jcalvarezlab committed:
Merge pull request #136 from JPDCjp/master

Add files via upload

BIGNATURE3 committed:
Add files via upload

2 trabajo relacionado con un alineamiento local. (Trabajo mandado por correo a Javier el sábado 4 de Marzo por no poder subir mi archivo a GitHub.)

Eled07 committed:
Add files via upload

JPDCjp committed:
Add files via upload

jcalvarezlab committed:
Merge pull request #141 from BIGNATURE3/master

Ejercicio2blastnlocal

jcalvarezlab committed:
Merge pull request #139 from Eled07/master

Add files via upload

dagomezs1 committed:
Create Prueba

dagomezs1 committed:
Add files via upload

dagomezs1 committed:
Delete Prueba

jcalvarezlab committed:
Merge pull request #142 from jfrancoc14/master

Add files via upload

jcalvarezlab committed:
Merge pull request #143 from sharajg/master

title

jfrancoc14 committed:
Add files via upload

sharajg committed:
Add files via upload

jcalvarezlab committed:
Merge pull request #144 from jfrancoc14/master

Add files via upload

sharajg committed:
Merge branch 'UniversidadEAFIT:master' into master

sharajg committed:
Add files via upload

jcalvarezlab committed:
Merge pull request #145 from sharajg/master

Ejercicio Shara

anamariavc committed:
Add files via upload

JPDCjp committed:
Merge branch 'UniversidadEAFIT:master' into master

JPDCjp committed:
Delete BlastAmanita.txt

JPDCjp committed:
Add files via upload

JPDCjp committed:
Delete sequence.fasta

AwkwardDCT committed:
Create danielacalle

dagomezs1 committed:
Create Alejandro_Blast

dagomezs1 committed:
Delete Alejandro_Blast

dagomezs1 committed:
Create prueba

dagomezs1 committed:
Add files via upload

dagomezs1 committed:
Delete prueba

dagomezs1 committed:
Delete Alejandro_Gomez directory

jcalvarezlab committed:
Merge pull request #148 from AwkwardDCT/patch-1

Create danielacalle

jcalvarezlab committed:
Merge pull request #149 from dagomezs1/master

Actividad Blast Alejandro Gómez

jcalvarezlab committed:
Create readme.rm

jcalvarezlab committed:
Add files via upload

jpmerchanc committed:
Add files via upload

jcalvarezlab committed:
Merge pull request #150 from jpmerchanc/master

Add files via upload

jcalvarezlab committed:
Merge pull request #146 from anamariavc/master

AnaMaria_trabajo2

jfrancoc14 committed:
Add files via upload

sharajg committed:
Add files via upload

jcalvarezlab committed:
Merge pull request #147 from JPDCjp/master

Tarea

jcalvarezlab committed:
Merge pull request #151 from jfrancoc14/master

Ejercicio Ensamblaje Juliana Franco Correa

jcalvarezlab committed:
Merge pull request #152 from sharajg/master

trabajofinal

dagomezs1 committed:
Add files via upload

mlopezg11 committed:
Add files via upload

Eled07 committed:
Add files via upload

jcalvarezlab committed:
Merge pull request #153 from dagomezs1/master

Add files via upload

jcalvarezlab committed:
Merge pull request #154 from mlopezg11/master

Add files via upload

jcalvarezlab committed:
Merge pull request #155 from Eled07/master

Add files via upload

JPDCjp committed:
Add files via upload

jcalvarezlab committed:
Merge pull request #156 from JPDCjp/master

Add files via upload

sgiraldos3 committed:
Add files via upload

AwkwardDCT committed:
Update danielacalle

AwkwardDCT committed:
Update danielacalle

AwkwardDCT committed:
Update danielacalle

AwkwardDCT committed:
Delete danielacalle

AwkwardDCT committed:
Merge branch 'UniversidadEAFIT:master' into master

AwkwardDCT committed:
Task 1 Daniela Calle

sgiraldos3 committed:
Add files via upload

AwkwardDCT committed:
Add files via upload

AwkwardDCT committed:
Add files via upload

sgiraldos3 committed:
Add files via upload

AwkwardDCT committed:
Add files via upload

sgiraldos3 committed:
Add files via upload

jcalvarezlab committed:
Merge pull request #159 from AwkwardDCT/master

Tarea 1 Daniela Calle

jcalvarezlab committed:
Merge pull request #157 from sgiraldos3/master

Add files via upload

jcalvarezlab committed:
Create Readme.rm

jcalvarezlab committed:
Create readme.rm

jcalvarezlab committed:
Add files via upload

jcalvarezlab committed:
Add files via upload

Proteome O. Sativa

jcalvarezlab committed:
Add files via upload

2025-2

jcalvarezlab committed:
Delete 2025/ClaseProgramacionTipodeDatos.ipynb

jcalvarezlab committed:
Add files via upload

jcalvarezlab committed:
Add files via upload

2026-1

Commit:
Reorganiza el curso de bioinformática para 2026

jcalvarezlab committed:
Merge pull request #160 from UniversidadEAFIT/refresh/course-2026

Reorganización del curso de bioinformática 2026
